pwa/src/components/logs/logTable/logTable.tsx

```diff
@@ -275,11 +261,12 @@ export const LogTable: React.FC<LogTableProps> = ({ id = null, query= null }) =>
                             return (<>
                               {log.requestContent
                                 ?
-                                <JsonCode body={log.requestContent} />
+                                <CodeBlock body={log.requestContent} />
```

Comment:
Er moet nog wel vanuit de backend een content en accept type als property toegevoegd worden op de Log zodat we daar op gecheckt kan worden voor xml of json

Reply:
Issue is created and default is json